README.md

```diff
@@ -5,5 +5,12 @@ Repo for Administrator
 1. If you do not have maven installed, you need to install it
 2. Import this project as a maven project
 3. Set up MySQL schema using the schema in src/test/resources/schema.sql
-4. Change root and password in src/main/resources/application.properties
-        to access your MySQL server with the library schema
+4. Create a file called `database-config.properties` in src/main/resources. Look at the code down below for an example. <br/>
+```
+spring.datasource.url = jdbc:mysql://hostname:port/data-base-name?useSSL=false&serverTimezone=UTC
+spring.datasource.username = username
+spring.datasource.password = password
+
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
I suggest making whitespace around the `=` consistent: in every other line it has a space before and after, but in this line there is none.